integration_test/src/lib.rs

```diff
@@ -71,12 +71,12 @@ mod test {
     fn integration_test() {
         trace::init();
         // metric::metric_init(PROMETHEUS_SCRAPE_PORT.parse::<SocketAddr>().unwrap());
-        let span = info_span!("integration test", auth = false, tls = false);
-        span.in_scope(|| {
-            info!("Running integration test");
-            let config = PravegaStandaloneServiceConfig::new(false, false, false);
-            run_tests(config);
-        });
+        // let span = info_span!("integration test", auth = false, tls = false);
```

Comment:
What's this about

Reply:
Fixed, was trying to only run bytestream for testing, forget to change it back